force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +313,8 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        Boolean boolStringSet = stringSet.addAll(inputList);
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
great work. I used basically the same solution.  you could return the set directly.  You don't need that boolStringset variable for anything 


  return new Set<String>(inputList);